frontend/src/scenes/error-tracking/errorTrackingDataNodeLogic.tsx

```diff
@@ -22,48 +22,48 @@ export const errorTrackingDataNodeLogic = kea<errorTrackingDataNodeLogicType>([
     })),
 
     actions({
-        mergeGroups: (indexes: number[]) => ({ indexes }),
```

Comment:
General question - what do the words "data node logic" mean here?

Reply:
Called it that because it connects to the "data node logic" from product analytics. A "node" is a term they use to describe queries e.g. query node. The "data" references the fact that the logic should have nothing to do with the UI